config/env.development

```diff
@@ -107,6 +107,11 @@ JWT_AUDIENCE=http://localhost
 # How long should a JWT work before it expires
 JWT_EXPIRES_IN=1h
 
+# Slack Bolt auth
+SLACK_BOT_TOKEN=
+SIGNING_SECRET=
```

Comment:
Because we have so many environment variables, and all 3 of these relate to this Slack Bot config, can we name them all `SLACK_BOT_*`:

```ini
SLACK_BOT_TOKEN=
SLACK_BOT_SIGNING_SECRET=
SLACK_BOT_CHANNEL_ID=
```

I think this will help later on as we maintain this code.

Reply:
For sure. Working on it.

Reply:
Resolved in ef681ba